pkg/workflows/sdk.go

```diff
@@ -0,0 +1,5 @@
+package workflows
+
+type Sdk interface {
```

Comment:
Should this be upper-case per go naming?
```suggestion
type SDK interface {
```